api/Document.json

```diff
@@ -9127,6 +9127,55 @@
           }
         }
       },
+      "selectstart_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/selectstart_event",
+          "description": "<code>selectstart</code> event",
+          "support": {
+            "chrome": {
+              "version_added": null
```

Comment:
This should be `false`. [Here's the source](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/dom/document.idl).